rest_server/RequestHandlerPut.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"SiodbIomgrProtocol"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/gin-gonic/gin"
+)
+
+func (restWorker RestWorker) putRow(c *gin.Context) {
```

Comment:
I'd just map PUT method to existing PATCH handler and not don't do any extra handler.